tools/grunt-tasks/minifyNames.js

```diff
@@ -1389,14 +1389,11 @@ function minifyNamesFn(grunt) {
                         // remove unused imports
                         removeUnusedImports(parsedFile);
             
-                        if (parsedFile.changed) {
-            
-                            var src = formatFile(parsedFile, theOptions.maxImportWidth);
-                            if (orgSrc !== src) {
-                                if (!theOptions.testOnly) {
-                                    // Rewrite the file
-                                    fs.writeFileSync(inputFile + theOptions.ext, src);
-                                }
+                        var src = formatFile(parsedFile, theOptions.maxImportWidth);
```

Comment:
This file (and the .gitignore) are the only manually changed file. All other changes in the PR where auto-generated from this script or the build system (ie rush)